projects/plugins/jetpack/tests/e2e/specs/forms/submission.test.js

```diff
@@ -0,0 +1,172 @@
+import { Plans, prerequisitesBuilder } from '_jetpack-e2e-commons/env/index.js';
+import { expect, test } from '_jetpack-e2e-commons/fixtures/base-test.js';
+import { BlockEditorPage } from '_jetpack-e2e-commons/pages/wp-admin/index.js';
+import playwrightConfig from '../../playwright.config.mjs';
+
+test.beforeAll( async ( { browser } ) => {
+	const page = await browser.newPage( playwrightConfig.use );
+	await prerequisitesBuilder( page )
+		.withCleanEnv()
+		.withLoggedIn( true )
+		.withWpComLoggedIn( true )
+		.withConnection( true )
+		.withPlan( Plans.Free )
+		.build();
+	await page.close();
+} );
+
+test.afterEach( async ( { requestUtils } ) => {
+	// List all feedback submissions.
+	// https://developer.wordpress.org/rest-api/reference/posts/#list-posts
+	const feedbackSubmissions = await requestUtils.rest( {
+		path: '/wp/v2/feedback',
+		params: {
+			per_page: 100,
+			// All possible statuses.
+			status: 'publish,future,draft,pending,private,trash',
+		},
+	} );
+
+	// Delete all feedback submissions one by one.
+	// https://developer.wordpress.org/rest-api/reference/posts/#delete-a-post
+	// "/wp/v2/feedback" does not yet support batch requests.
+	await Promise.all(
+		feedbackSubmissions.map( feedback =>
+			requestUtils.rest( {
+				method: 'DELETE',
+				path: `/wp/v2/feedback/${ feedback.id }`,
+				params: {
+					force: true,
+				},
+			} )
+		)
+	);
+} );
+
+test.describe( 'Forms: Submission', () => {
+	test( 'Submits a simple contact form', async ( { editor, page } ) => {
+		const formTitle = 'E2E Test Form';
+		await test.step( 'Visit the block editor and insert a form', async () => {
+			const blockEditor = await BlockEditorPage.visit( page );
+			await page.waitForURL( '**/post-new.php' );
+			await blockEditor.waitForEditor();
+			await blockEditor.closeWelcomeGuide();
```

Comment:
Instead of using `page`, why not simply `await admin.createNewPost();`?

`admin` comes from the same source as `page` here.

Reply:
I copied it from another test, but yeah, that's a good idea!

Reply:
Done 👍